src/library/crate/cratestorage.h

```diff
@@ -287,6 +290,8 @@ class CrateStorage: public SqlStorage {
             CrateId crateId) const;
     CrateTrackSelectResult selectTrackCratesSorted(
             TrackId trackId) const;
+    CrateTrackSelectResult selectTracksSortedByCrateNameLike(
```

Comment:
In the interest of minimizing potential merge conflicts with #1117, I think we should hold off on renaming things for now.